modules/openy_features/openy_node/modules/openy_node_social_post/config/install/social_feed_fetcher.settings.yml

```diff
@@ -0,0 +1,27 @@
+facebook_enabled: true
+fb_page_name: null
+fb_app_id: ''
+fb_secret_key: null
+fb_no_feeds: 10
+instagram_enabled: true
+in_client_id: null
+in_redirect_uri: ''
+in_auth_link: ''
+in_access_token: null
+in_picture_count: 10
+in_picture_resolution: standard_resolution
+in_post_link: true
+twitter_enabled: true
+tw_consumer_key: null
+tw_consumer_secret: null
+tw_access_token: null
+tw_access_token_secret: null
+tw_count: 10
+last: ''
+submit: 'Run cron now'
+social_feed_fetcher_interval: '60'
+formats_post_format: full_html
+form_build_id: form-S9JcG1m0ZgyeoB9dCz160pX8ec7KBPy4N2EuEXO01iY
```

Comment:
looks like a bug
```form_build_id``` is a dynamic value, why stored in config?

Reply:
fixed